openfe/protocols/openmm_rbfe/_rbfe_utils/multistate.py

```diff
@@ -115,8 +116,7 @@ class creation of LambdaProtocol.
             context, context_integrator = context_cache.get_context(
                                              compound_thermostate_copy)
             # TODO: move to our own MD tasks
-            # feptasks.minimize(compound_thermodynamic_state_copy,
-            #                   sampler_state)
+            #minimize(compound_thermostate_copy, sampler_state)
```

Comment:
The original perses code does the minimization here, however in my opinion it's a bit problematic to have it hidden here. It would be better for this to be a) controlled, b) easy to spot in the protocol.